pom.xml

```diff
@@ -29,6 +29,10 @@
     <tag>HEAD</tag>
   </scm>
 
+  <properties>
+    <dep.kafka.version>0.10.1.0</dep.kafka.version>
+  </properties>
+
```

Comment:
I'm not sure how we handle this.  The Chimera cluster is on 0.10
Maybe we will have to have two builds of otj-kafka, for each Kafka version :(